komodo/__init__.py

```diff
@@ -6,6 +6,8 @@
     try:
         from ._version import version as VERSION
     except ImportError:
-        msg = "Failed to find autogenerated _version.py."
+        msg = (
+            "Failed to find autogenerated _version.py."  # pylint: disable=invalid-name
+        )
```

Comment:
I would just skip the temporary variable here and just to `ImportError("Failed to ...")`